### ISSUE TITLE
Result value of assignment wrong

### ISSUE THREAD

**Issue by KimBruce** [gracelang/minigrace#155]
The results type of an assignment statement is supposed to be done, but it apparently is the value on the right side of the assignment:

```
dialect "objectdraw"
var color: Color

def v = if (true) then {
    color := red
} else {
    color := blue
}

print(color)
print(v)
```
The output is
```
color w/ rgb(255, 0, 0)
color w/ rgb(255, 0, 0)
```
but the value of v should be done.

**Comment by apblack** [comment]
I can't reproduce this.  The following program
```
var color: String

def v = if (true) then {
    color := "red"
} else {
    color := "blue"
}

print "color = {color}"
print "v == {v}"
```
produces
```
color = red
v == done
```
in both the C and JS backends.  

This bag has existed in the past, but was fixed on 9th September 2015 in commit ce3b44ed277a (generation 2736)

**Comment by KimBruce** [comment]
Thanks, I didn't think to check it on your installation. I haven't updated
my web page with the compiler since the semester started (for fear of
running into new errors).
